hack/istio/install-bookinfo-demo.sh

```diff
@@ -211,15 +226,34 @@ $CLIENT_EXE get services -n ${NAMESPACE}
 $CLIENT_EXE get pods -n ${NAMESPACE}
 
 # If OpenShift, we need to do some additional things
-if [[ "$CLIENT_EXE" = *"oc" ]]; then
+if [[ "${IS_OPENSHIFT}" = "true" ]]; then
   $CLIENT_EXE expose svc/productpage -n ${NAMESPACE}
   $CLIENT_EXE expose svc/istio-ingressgateway --port http2 -n ${ISTIO_NAMESPACE}
-  cat <<NAD | $CLIENT_EXE -n ${NAMESPACE} create -f -
+  if [[ "${IS_MAISTRA}" == "true" ]]; then
+    cat <<NAD | $CLIENT_EXE -n ${NAMESPACE} create -f -
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
   name: istio-cni
 NAD
+  fi  
+  cat <<SCC | $CLIENT_EXE -n ${NAMESPACE} create -f -
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: bookinfo-scc
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+users:
+- "system:serviceaccount:${NAMESPACE}:bookinfo-details"
+- "system:serviceaccount:${NAMESPACE}:bookinfo-productpage"
+- "system:serviceaccount:${NAMESPACE}:bookinfo-ratings"
+- "system:serviceaccount:${NAMESPACE}:bookinfo-ratings=v2"
+- "system:serviceaccount:${NAMESPACE}:bookinfo-reviews"
```

Comment:
this is fine. If they don't exist, they should just be ignored IIRC. But what about things like mongo and mysql? Do those have SAs, too?

Reply:
I didn't see them.  Should I add default as well, just to cover the bases?

Reply:
I wouldn't add "default" unless you know you need it.

Reply:
I see services in all the demos that don't have a serviceaccount, so I think we may need it